ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/connected/distance/AdjacentLdesFragmentFilter.java

```diff
@@ -0,0 +1,27 @@
+package be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.connected.distance;
+
+import be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.Tile;
+import be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.converter.TileConverter;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.valueobjects.LdesFragment;
+
+import java.util.function.Predicate;
+
+public class AdjacentLdesFragmentFilter implements Predicate<LdesFragment> {
+
+    private final LdesFragment originLdesFragment;
+
+    public AdjacentLdesFragmentFilter(LdesFragment ldesFragment) {
+        this.originLdesFragment = ldesFragment;
+    }
+
+    @Override
+    public boolean test(LdesFragment ldesFragment) {
```

Comment:
would come up with a better name ;)

Reply:
public boolean test(....) is een method die ik override van het Predicate<>: https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html

Agree that it would be more convenient to specify a more relevant name.
Will add a javadoc to explain.